docs/development.rst

```diff
@@ -0,0 +1,174 @@
+Development
+===========
+
+The theme developers follow the guidelines below for development and release
+planning. Documentation authors can decide which theme release works best for
+their project based on required browser/operating system combinations or
+dependency support.
+
+Supported browsers
+------------------
+
+Official browser support is determined by the most popular browser/operating
+system combinations in our pageview analytics. Officially supported combinations
+are commonly tested during development, and are always tested before final
+release of a new version of the theme.
+
+Older releases of supported combinations, and some less common combinations, are
+considered partially supported. This means that we do not consistently test
+these combinations, however we do expect user experience to be comparable with
+supported combinations.
+
+.. csv-table:: Supported browser combinations
+    :widths: 6, 12, 4
+    :header-rows: 1
+    :file: supported-browsers.csv
+
+.. versionadded:: 1.0
+    Supported browser and operating system combinations added
+
+There are several browser/operating system combinations that are not supported
+by the theme developers at all. Unsupported combinations do not receive testing
+or development, and we likely won't accept major contributions for these
+combinations.
+
+Unsupported browser/operating system combinations include:
+
+Internet Explorer (any OS, versions <=10)
+    **Unsupported.** IE11 is the last partially supported version. We do no
+    testing on prior versions.
+
+Internet Explorer (any OS, version 11)
+    We currently only partially support IE11, and only test for major bugs.
+    Support will be removed in the 2.0 release.
+
+Opera (any OS, any version)
+    **Community support only.** We do not receive enough traffic with this
+    browser to officially support it in testing and development.
+
+Supported dependencies
+----------------------
+
+The theme officially supports the following dependencies in your Sphinx project:
+
+.. list-table:: Supported dependencies
```

Comment:
This feels like we should have it in the README, not hidden away in a page only for developers. I think we want this to be front & center for all users.

Reply:
I'll link it from the readme